modal.css

```diff
@@ -0,0 +1,251 @@
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.3);
+  box-sizing: border-box;
+}
+
+.pop-modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+  border-radius: 10px;
+}
+
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+  width: 40px;
+  height: 40px;
+  background: #ebecf0;
+  mix-blend-mode: multiply;
+  border-radius: 8px;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 8px;
+  cursor: pointer;
+}
+
+.close:hover,
+.close:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+  transform: scale(1, 1);
+}
+
+.desktop-img {
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
- [OPTIONAL] Your modal looks great, but if the user's browser viewport is large, the image can become so large that pushes the content of the modal out of the screen, and the user needs to scroll down to see it. This may not be a good user experience and would be nice if the screen can have a limited size so the modal can fit inside the user browser view.
 -Example:
<img width="1392" alt="image" src="https://user-images.githubusercontent.com/61919617/190184485-08cd76f8-5d35-4fb5-a1fd-418476b69d27.png">